NYTimes.com.js

```diff
@@ -37,8 +37,18 @@
 
 
 function detectWeb(doc, url) {
-	if (getSearchResults(doc, true)) {
-		return "multiple";
+	//we use another function name to avoid confusions with the
```

Comment:
So EM is exporting detectWeb? I thought translators only exported specific methods- and they would be bound to the exported translator object, not in the global scope. But it's been ages...